awsecs/awsecs.go

```diff
@@ -154,6 +163,13 @@ func updateTaskDef(taskDefARN, gitsha string, ecsClient ecsiface.ECSAPI) (update
 			previousGitsha = t[len(t)-1]
 		}
 
+		// Only update if we find an existing image that is different from the new gitsha
```

Comment:
this is good today, but eventually when we add sidecars, they will not have the same tag (git-SHA) so this may need some changes. 

we have resorted to using `service*` as the name for our containers; so we can use that OR skip anything like `envoy` or `xray` ... but we'll see later.